README.markdown

```diff
@@ -31,7 +31,8 @@ This should give you the flexibility you need for customizing your input/output
 
 ### Building Your Phar
 
-Simply run `php package.php`, which will create the file: `build/doctrine-migrations.phar` for you.
+The PHAR is ready for [download here](http://github.com/downloads/ericclemmons/migrations/doctrine-migrations.phar), but if you still would like to build your PHAR
```

Comment:
Given that the URL you are using is referencing the github downloads feature which has been shutdown more than 1 year ago, I'm quite sure the phar is totally outdated. So this looks like a bad idea to use it